git_details.env

```diff
@@ -0,0 +1 @@
+    PAT_1="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"
```

Comment:
You should immediately remove this token and revoke this token from your github settings, someone could get access to your github profile. @johnbabu021